### ISSUE TITLE
Unminimized distribution lacks vendor specific selector properties

### ISSUE THREAD

**Issue by snovak7** [dynamicweb/swiffy-slider#76]
With version 1.6.0 and probably less... I stumbled upon that icons are not shown in latest Brave, Chromium based browser... mask isn't prefixed with -webkit- as with minimized version in "distribution" output.

Keep up the good job 👍

**Comment by nicped** [comment]
Good catch - can see that the dist/css/swiffy-slider.css does not contain the prefix - that file is just the original and not parsed with autoprefixer as the min version.
Will look into a solution.
